freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -66,11 +71,85 @@ def handle(self, event):
             "Orignial nvrs of build in the advisory #{0} are: {1}".format(
                 db_event.search_key, " ".join(original_nvrs)))
 
-        msg = "Skipping due to being blocked on further implementation for now."
+        index_images = self._pyxis.get_operator_indices()
+        # get latest bundle images per channel per index image filtered by higher version
+        all_bundles = self._pyxis.get_latest_bundles(index_images)
```

Comment:
I would run this after `get_digests_by_nvrs` since `get_digests_by_nvrs` should be much faster and the output could cause the handler to exit prematurely.